src/helpers/stats.js

```diff
@@ -46,6 +46,7 @@ export default {
                 stats.answered--;
             } else if ((isItemCurrentlyAnswered || sync) && !item.answered) {
                 stats.answered++;
+                // eslint-disable-next-line no-dupe-else-if
```

Comment:
ESLint says the final `else if` block can't ever be reached, but I don't think it takes `sync` into account.

Reply:
it looks like eslint is right. To reach the 3rd condition, `isItemCurrentlyAnswered` should be falsy and `item.answered` should be truthy, but with that you'll always reach the 1st block `if (!isItemCurrentlyAnswered && item.answered) {` 
Anyway, it's safer to deactivate the rule.